kong/llm/drivers/shared.lua

```diff
@@ -190,6 +261,7 @@ end
 
 function _M.post_request(conf, response_string)
   if conf.logging and conf.logging.log_payloads then
+    kong.log.warn(response_string)
```

Comment:
Is this some debug that got left in?

If not, I think this should be debug level.